vllm/config.py

```diff
@@ -1797,13 +1797,6 @@ def __post_init__(self):
                 "'otlp_traces_endpoint'. Ensure OpenTelemetry packages are "
                 f"installed. Original error:\n{otel_import_error_traceback}")
 
-        if ((self.collect_model_forward_time
```

Comment:
Not sure why this was added. OpenTelemetry is independent of compute time measure; we don't say we can't measure compute time without OpenTelemetry.